.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -152,7 +152,7 @@ module backend_service {
   api_url               = local.backend_url
   frontend_url          = local.frontend_url
   remote_dev_prefix     = local.remote_dev_prefix
-  health_check_path     = "/health"
+  health_check_path     = "/v2/health/"  # the trailing slash here is vital.
```

Comment:
it looks like something went wrong with your merge?

Reply:
i think this is because i merged in my warnings branch and trunk, i would just ignore this as this file won't get overwritten on trunk as the commits exist on this branch and trunk and the files are identical.